package.json

```diff
@@ -13,7 +13,7 @@
   ],
   "scripts": {
     "pretest": "npm run build",
-    "test": "API_BASE_URL=http://localhost:1234/ TEST_PORT=1234 TS_NODE_CACHE=0 nyc mocha",
+    "test": "TEST_PORT=1234 TS_NODE_CACHE=0 nyc mocha",
```

Comment:
TEST_PORT is not used in the code

Reply:
`TEST_PORT` is still used in middleware test.